.travis.yml

```diff
@@ -22,7 +22,6 @@ before_script:
 - bundle exec rake webdrivers:chromedriver:update
 script:
 - xvfb-run -a bundle exec rspec
-before_deploy: "./compile-build"
```

Comment:
We need this to create the `release.tar.gz`

Reply:
And this one.